### PR TITLE
Update ru-RU translation

### PR DESCRIPTION
**Pull request by admik**
Updated Russian translation.

### CODE REVIEW

**Comment by Bwko** [comment]
@denji Could you take a look at this?

**Comment by tboerger** [comment]
This review is totally up to the native speaker or people that can at least verify this is correct :)

**Comment by anton-dutov** [comment]
I'm native russian speaker and i not sure for some translations
"issues" translated as "tasks" by myself that's wrong - "issue" most common mean nearest to "troubles" than to "task"
Other notes in review

File: options/locale/locale_ru-RU.ini [hunk 1]
```diff
@@ -1198,3 +1206,9 @@ invalid_input_type=Вы не можете загружать файлы этог
 file_too_big=Размер файла ({{filesize}} МБ) больше чем максимальный размер ({{maxFilesize}} МБ).
 remove_file=Удалить файл
 
+[notification]
+notifications=Уведомления
+unread=Не прочтенные
+read=Прочтенные
+no_unread=У вас нет не прочтенных уведомлений.
```
Comment (anton-dutov) [inline options/locale/locale_ru-RU.ini]: Use "Нет непрочитанных уведомлений"

File: options/locale/locale_ru-RU.ini [hunk 2]
```diff
@@ -1198,3 +1206,9 @@ invalid_input_type=Вы не можете загружать файлы этог
...
+no_read=У вас нет прочтенных уведомлений.
```
Comment (anton-dutov) [inline options/locale/locale_ru-RU.ini]: Use "Нет прочитанных уведомлений"

File: options/locale/locale_ru-RU.ini [hunk 3]
```diff
@@ -1198,3 +1206,9 @@ invalid_input_type=Вы не можете загружать файлы этог
 file_too_big=Размер файла ({{filesize}} МБ) больше чем максимальный размер ({{maxFilesize}} МБ).
 remove_file=Удалить файл
 
+[notification]
+notifications=Уведомления
+unread=Не прочтенные
```
Comment (anton-dutov) [inline options/locale/locale_ru-RU.ini]: Better is "Непрочитанные"

File: options/locale/locale_ru-RU.ini [hunk 4]
```diff
@@ -1198,3 +1206,9 @@ invalid_input_type=Вы не можете загружать файлы этог
...
+read=Прочтенные
```
Comment (anton-dutov) [inline options/locale/locale_ru-RU.ini]: Same as previous "Прочитанные"

**Comment by admik** [comment]
@anton-dutov @denji  I've changed translation according to your suggestions.

**Comment by kuntashov** [comment]
@anton-dutov @admik 
"Проблема" is wrong translation for "issue"!
No one issue tracking software uses such translation. All of them translates "issue" into russian "Задача". It's common russian term for english "issue" for many popular issue tracking and project management software. 

Some examples:

* Mantis BT - https://github.com/mantisbt/mantisbt/blob/master/lang/strings_russian.txt#L1092
* Redmine - https://github.com/redmine/redmine/blob/master/config/locales/ru.yml#L530
* Bitbucket - https://www.transifex.com/atlassian/bitbucketorg/translate (need login to view translation)

All the more "issue" does not mean "troubles" in genereal. It means "an important question that is discussed and must be settled". See https://www.vocabulary.com/dictionary/issue

**Comment by admik** [comment]
@anton-dutov @kuntashov  @denji @Bwko There is no exact translation for “issue” into Russian. Most tracking system translate it as a “task”, some as “request” (Atlassian Jira). I prefer translate it as a “task”, but I believe that translation as a “trouble” is ok. Should we vote?

**Comment by asosnoviy** [comment]
А мне больше нравится "обсуждение".

**Comment by denji** [comment]
@asosnoviy +1

**Comment by Lourens-Rich** [comment]
Оставьте как "Задачи" и не выпендривайтесь.

**Comment by anton-dutov** [comment]
Предочитающие"задачи" побеждают, за сим извиняюсь внес смуту =)

**Comment by Bwko** [comment]
@admik Any updates on this? Do you need to change anything or is it good to go?

**Comment by admik** [comment]
@Bwko I've changed translation of 'issue' back to 'task', because this translation is being used in most tracking systems. Now it is good to go.

**Comment by Bwko** [comment]
LGTM

File: options/locale/locale_ru-RU.ini [hunk 5]
```diff
@@ -303,7 +309,7 @@ add_email_success=Новый адрес электронной почты усп
 manage_ssh_keys=Управление SSH ключами
 add_key=Добавить ключ
 ssh_desc=Это список ключей SSH связанных с вашей учетной записью. Удаляйте любые неизвестные вам ключи.
-ssh_helper=<strong>Нужна помощь?</strong> Ознакомьтесь с нашим путеводителем по <a href="%s">созданию SSH-ключей</a> или посмотрите решения <a href="%s">частых проблем, связанных с SSH</a>.
+ssh_helper=<strong>Нужна помощь?</strong> Ознакомьтесь с нашим путеводителем по <a href="%s">созданию SSH-ключей</a> или посмотрите решения <a href="%s">частых задач, связанных с SSH</a>.
```
Comment (makhov) [inline options/locale/locale_ru-RU.ini]: Может "ответы на частые вопросы по ssh"?
А то фраза не по-русски звучит

**Review by makhov — CHANGES_REQUESTED** [review]

File: options/locale/locale_ru-RU.ini (hunk 5 above)
Comment (n-dimens) [reply options/locale/locale_ru-RU.ini]: Мне кажется, в данном случае, проще\лучше будет все-таки "решения частых проблем" (в английском варианте так и идет - problems). А вот "путеводителем", есть мнение, лучше заменить на "руководством" (как-то ближе к технической терминологии).

Comment (makhov) [reply options/locale/locale_ru-RU.ini]: Согласен, так даже лучше

Comment (n-dimens) [reply options/locale/locale_ru-RU.ini]: @admik Что думаешь по этому поводу, как автор PR?

Comment (admik) [reply options/locale/locale_ru-RU.ini]: @svgorbunov I support your suggestion. It sounds much better.

**Comment by makhov** [comment]
LGTM

**Review by denji — APPROVED** [review]

**Comment by lunny** [comment]
let L-G-T-M work

**Comment by lunny** [comment]
LGTM

File: options/locale/locale_ru-RU.ini [hunk 6]
```diff
@@ -136,8 +138,12 @@ issues.in_your_repos=В ваших репозиториях
 
 [explore]
 repos=Репозитории
+organizations=Организации
 users=Пользователи
 search=Поиск
+repo_no_results=Не найдено ни одного репозитория.
```
Comment (admik) [inline options/locale/locale_ru-RU.ini:144]: @denji Your proposal to translate "There are no matched repositories found" as "Нет данных репозитория" in Russian means "There are no data in the repository". It is not correct.

File: options/locale/locale_ru-RU.ini [hunk 7]
```diff
@@ -136,8 +138,12 @@ issues.in_your_repos=В ваших репозиториях
...
+user_no_results=Не найдено ни одного пользователя.
```
Comment (admik) [inline options/locale/locale_ru-RU.ini:145]: @denji Your proposal to translate "There are no matched users found" as "Нет данных пользователя" in Russian means "There are no data about user". It is not correct.

File: options/locale/locale_ru-RU.ini [hunk 8]
```diff
@@ -136,8 +138,12 @@ issues.in_your_repos=В ваших репозиториях
...
+org_no_results=Не найдено ни одной организации.
```
Comment (admik) [inline options/locale/locale_ru-RU.ini:146]: @denji The same as previous.

File: options/locale/locale_ru-RU.ini [hunk 9]
```diff
@@ -550,6 +556,8 @@ issues.label_modify=Изменение метки
 issues.label_deletion=Удаление метки
 issues.label_deletion_desc=Удаление ярлыка затронет все связанные задачи. Продолжить?
 issues.label_deletion_success=Метка была удалена успешно!
+issues.label.filter_sort.alphabetically=По алфавиту
+issues.label.filter_sort.reverse_alphabetically=С конца алфавита
```
Comment (admik) [inline options/locale/locale_ru-RU.ini:560]: @denji "Обратно по названию" doesn't sound good in Russian. "С конца алфавита" sounds much better.

Comment (kuntashov) [reply options/locale/locale_ru-RU.ini:560]: Usually "alphabetically" is translated as "По возрастанию", and "Reverse alphabetically" as "По убыванию".

**Pull request merged**